pkg/internal/tests/cross-tests/configure.go

```diff
@@ -141,8 +144,11 @@ func Configure(
 
 	pt.Up(t)
 
-	require.True(t, tfResult.wasSet, "pulumi configure result was not set")
-	require.True(t, tfResult.wasSet, "pulumi resource result was not set")
```

Comment:
Did you only mean to delete one of the tfResult.wasSet?

Reply:
We had two identical copy paste checks, both were wrong:

1. One should have been an assert that `puResult.wasSet` was true (added on line +147).
2. The other I meant to delete.